design/design-doc-stacks.md

```diff
@@ -1,15 +1,21 @@
-# Crossplane Extensions
+# Crossplane Stacks
 * Owner: Jared Watts (@jbw976)
 * Reviewers: Crossplane Maintainers
-* Status: Accepted, revision 1.0
+* Status: Accepted, revision 1.1
 
-This document aims to provide details about the experience and implementation for Crossplane “extensions”, which can add new functionality/support, types, and controllers to Crossplane.
+This document aims to provide details about the experience and implementation for Crossplane “Stacks”, which can add new functionality/support, types, and controllers to Crossplane.
+
+## Revisions
+
+* 1.1
+  * Renamed Extensions concept to Stacks (`Extension` code references are unaffected) [#571](https://github.com/crossplaneio/crossplane/issues/571)
+  * Added additional Questions and Open Issues
 
 ## Experience
 
 The core experience for consuming new functionality in Crossplane is composed of 2 steps:
 
-1. Create an extension request for the name of the extension or one of the CRDs that it owns
+1. Create an extension request for the name of the Crossplane Stack or one of the CRDs that it owns
```

Comment:
Is "extension request" still the preferred terminology for installing a Stack?

Reply:
A design-doc-only change leaves some of these questions open, I feel.

This set of changes reflects the transition in terminology as a precursor to any code changes.

I don't want to confuse anyone reading this and then comparing notes against the code where types and kinds are still called `Extension`.  It was my intent to leave `Extension` and `ExtensionRequest` references alone for now, even if it is only for a few days/weeks.

Reply:
I'll reword this to "installation request" to disambiguate it from the legacy Extensions while preserving the same meaning.